src/common/translation/i18n/fi.json

```
@@ -480,17 +480,11 @@
       }
     },
     "preliminaryForm": {
-      "childBirthdate": {
+      "childBirthyear": {
         "input": {
-          "day": {
-            "placeholder": "pv"
-          },
-          "label": "Lapsen syntymäaika (pp.kk.vvvv)",
-          "month": {
-            "placeholder": "kk"
-          },
+          "label": "Lapsen syntymäaika (vvvv)",
```

```suggestion
          "label": "Lapsen syntymävuosi",
```
i.e. "Child's birthyear" in Finnish